Doc/whatsnew/3.7.rst

```diff
@@ -604,3 +604,21 @@ CPython bytecode changes
 
 * Added two new opcodes: :opcode:`LOAD_METHOD` and :opcode:`CALL_METHOD`.
   (Contributed by Yury Selivanov and INADA Naoki in :issue:`26110`.)
+
+
+Documentation
+=============
+
+.. _whatsnew37-pep545:
+
+PEP 545: Python Documentation Translations
+------------------------------------------
+
+There's now a PEP describing how the Python documentation is
+translated, and two translation has been added:
+
+- Japanese: https://docs.python.org/ja/ https://github.com/python/python-docs-ja
+- French: https://docs.python.org/fr/ https://github.com/python/python-docs-fr
```

Comment:
Please add labels to link. Like `French documentation <...>`_ (`French GitHub project <...>`_).

Reply:
My intention was to show the URLS, not only to give links. The URLs themseve are an information in this context, from my point of view. But the documentation link may be a more important information than the github URL, and having a square of four URLs may be huge, why not:

```
- Japanese: https://docs.python.org/ja/ (`repository <...>`_)
- French: https://docs.python.org/fr/ (`repository <...>`_)
```